assets/css/_pages.scss

```diff
@@ -1,3 +1,33 @@
 span.highlight {
     background-color: yellow;
 }
+
+.historical-records {
+  table {
+    border-collapse: collapse;
+    margin: $small-spacing 0;
+    table-layout: fixed;
+    width: auto;
+  }
+
+  table:first-child {
+    width: 100%;
+  }
+
+  th {
+    border: 1px solid shade($base-border-color, 25%);
+    font-weight: 600;
+    padding: $small-spacing;
+    text-align: left;
+    background-color: #333;
+    color: transparentize(white, 0.3);
+  }
+
+  td {
+    border: $base-border;
```

Comment:
Properties should be ordered padding, font-weight, text-align, border

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)